packages/flutter_tools/BUILD.gn

```diff
@@ -58,6 +58,84 @@ dart_tool("fuchsia_asset_builder") {
 
   disable_analysis = true
 
+  sources = [
```

Comment:
A DBC here -

Are things going to break downstream if a file is added to flutter_tools but not to this list? If so, I'd recommend adding a tool to this repo that can re-generate this portion of the gn script in place, and a CI step on the bots that will fail if the list is not up-to-date. Both should be quick to do.

Reply:
Yes, but that is already the case before this change.